src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -94,6 +94,25 @@ public function storeHasAProductPricedAt($productName, $price = 0)
     }
 
     /**
+     * @Given this product has :variantName variant priced at :price
+     */
+    public function productHasAVariantPricedAt($variantName, $price)
```

Comment:
It is added in #4353. Price could be transformed to int thanks to lexical transformer